### DISCUSSION TITLE
How can I reisntall the OS and keep my files?

### DISCUSSION THREAD

**Discussion by traxap** [Joshua-Riek/ubuntu-rockchip#556] [Q&A]
Can I reinstall the OS and keep my Home files?  I tend to experiment with different apps and when the system becomes unstable I would like to be able to reinstall rather than reformat, re-partition and install a fresh OS, is it possible?

Thank you.

**Comment by Joshua-Riek** [comment]
you can copy your /home folder to some USB drive, and use that for recovery. Or you can make a clone of your currently running system with the command `sudo ubuntu-rockchip-install /dev/sdX` where /dev/sdX is a USB or hard drive. This command will copy the currently running system to another drive.

**Comment by traxap** [comment]
Thank you Josh for your quick reply and suggestions.
